packages/Chem/src/tests/sketcher-tests.ts

```diff
@@ -97,13 +102,21 @@ async function testSmarts(rdkitModule: any, funcs: DG.Func[]) {
   qmol?.delete();
 }
 
-async function testSmiles(rdkitModule: any, funcs: DG.Func[], input?: boolean) {
+const validationFunc = (s: string) => {
+  const valFunc = DG.Func.find({package: 'Chem', name: 'validateMolecule'})[0];
+  const funcCall: DG.FuncCall = valFunc.prepare({s});
```

Comment:
Why are we not using the autogenerated API for the library?

Reply:
Not sure that I got question correctly. I had to call it that way because otherwise rdkit module inside validation function wan not initializing.

Reply:
OK. Proceed further this time, it appears that most people don't know about the autogenerated API... will educate the team later.